File: vortexdbg-api/src/main/java/com/vortexdbg/unwind/Frame.java
package com.vortexdbg.unwind;

import com.vortexdbg.pointer.VortexdbgPointer;

public class Frame {

    public final VortexdbgPointer ip, fp;

    public Frame(VortexdbgPointer ip, VortexdbgPointer fp) {
        this.ip = ip;
        this.fp = fp;
    }

    final boolean isFinish() {
        return fp == null;
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/unwind/Frame.kt
package com.vortexdbg.unwind

import com.vortexdbg.pointer.VortexdbgPointer

class Frame(@JvmField val ip: VortexdbgPointer?, @JvmField val fp: VortexdbgPointer?) {

    fun isFinish(): Boolean {
        return fp == null
    }

}
